runtime/near-vm-logic/src/logic.rs

```diff
@@ -108,7 +108,6 @@ impl<'a> VMLogic<'a> {
         memory: &'a mut dyn MemoryLike,
         current_protocol_version: ProtocolVersion,
     ) -> Self {
-        ext.reset_touched_nodes_counter();
```

Comment:
why?

Reply:
Because if I want to track nodes counter before and after receipt processing, resetting counter in the middle of execution leads to incorrect results.